state/wwwpublic/status-events.json

```diff
@@ -1,4 +1,26 @@
 [
+
+  {
+    "status": "AUTO",
+    "work_start": "2025-01-01T18:00:00+01:00",
+    "systems": [
+        "ERDA"
+    ],
+    "announce_start": "2024-12-20T08:00:00+01:00",
+    "title": {
+      "EN": "Planned systems maintenance on ERDA",
+      "DK": "Planlagt systemvedligehold på ERDA"
+    },
+    "services": [
+        "ALL"
+    ],
+    "work_end": "2025-01-02T10:00:00+01:00",
+    "description": {
+        "EN": "General system upgrades, and final transitioning to new authentication service OpenID Connect.",
+        "DK": "Generelle systemopdateringer, samt endelig overgang til ny autentikeringstjeneste OpenID Connect."
```

Comment:
A small typo in "autentikeringstjeneste" should properly be "autentificeringstjeneste"